utilfw/utilfw.go

```diff
@@ -0,0 +1,36 @@
+package utilfw
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// ValueToSetType ensures we have a types.Set literal
+func ValueToSetType(v attr.Value) types.Set {
+	if vb, ok := v.(types.Set); ok {
+		return vb
+	} else if vb, ok := v.(*types.Set); ok {
```

Comment:
If you have a `return` from an `if` you don't need an `else`, you just need 3 `if-return`

Reply:
yes, agree, thanks!